src/sql/workbench/contrib/scripting/browser/scripting.contribution.ts

```diff
@@ -256,7 +256,27 @@ MenuRegistry.appendMenuItem(MenuId.ExplorerWidgetContext, {
 		id: commands.ExplorerScriptSelectAction.ID,
 		title: commands.ExplorerScriptSelectAction.LABEL
 	},
-	when: ContextKeyExpr.or(ItemContextKey.ItemType.isEqualTo('view'), ItemContextKey.ItemType.isEqualTo('table')),
+	when:
+		ContextKeyExpr.and(
+			ItemContextKey.ConnectionProvider.notEqualsTo('kusto'),
+			ContextKeyExpr.or(
+				ItemContextKey.ItemType.isEqualTo('view'),
+				ItemContextKey.ItemType.isEqualTo('table')
+			)
+		),
+	order: 2
+});
+
+MenuRegistry.appendMenuItem(MenuId.ExplorerWidgetContext, {
+	command: {
+		id: commands.ExplorerScriptSelectAction.ID,
+		title: 'Take 10'
```

Comment:
This should be localized (and like the above the action class itself should have a LABEL property with this value)

Reply:
good catch, I was too focused on the when clause syntax. 🤣

Reply:
Sure, made the change